TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/managers/CoreInstanceManager.java

```diff
@@ -439,21 +439,25 @@ public void processInstanceDemands() {
         stopUnneededProxies();
 
         Queue<GroupInstanceDemand> demands = new PriorityQueue<>();
+        Queue<GroupInstanceDemand> staticDemands = new PriorityQueue<>();
 
         for (Group group : getGroups()) {
             int amount = needed(group);
             if (amount <= 0) continue;
-            if (group.isStatic()) demands.add(new GroupInstanceDemand(group, 1));
+            if (group.isStatic()) staticDemands.add(new GroupInstanceDemand(group, 1));
             else demands.add(new GroupInstanceDemand(group, amount));
         }
-        while (!demands.isEmpty()) {
-            GroupInstanceDemand demand = demands.poll();
+
+        while (!staticDemands.isEmpty()) { // Start static instances first
```

Comment:
Why would we want to start the static demands first? Can't we simply assign them a lower priority than the dynamic demands?

Reply:
I just undid my changes but I think it would be the best if we start all servers depending its priority